docs/docs/user/subscribing.md

```diff
@@ -41,7 +41,7 @@ All options:
 
 Option                               | Description                                     | Default value
 ------------------------------------ | ----------------------------------------------  | -------------
-trackingMode                         | track incoming messages                         | TRACKING_OFF
+trackingMode                         | track incoming -> outgoing messages             | trackingOff
```

Comment:
@adididas122 ups this is not what I had in mind, I meant "track outgoing messages" as the final description :)

Reply:
fixed